Assets/Scripts/BlockBehaviour.cs

```diff
@@ -15,6 +15,8 @@ public class BlockBehaviour : MonoBehaviour
     public BlockFaceBehaviour Left;
     public BlockFaceBehaviour Right;
 
+    public LayerMask IgnoreCollision;

```

Comment:
same here

Reply:
For this, a raycast that fires out from a block will ignore anything selected in the layer mask. In the inspector, i have selected for it to ignore `Player`. The block will push players.

Reply:
ok. so it needs to be set for each block which we place in a level? will we ever need it to ignore more than 1 layer?

Reply:
We can set more than 1 layer in the layer mask. At the same time, the ignore can be set at the prefab level, to affect all blocks.

Reply:
kk, sounds good 👍